python/eos/plot/plotter.py

```diff
@@ -63,6 +64,34 @@ def next_color(self):
         self.__next_color = (self.__next_color + 1) % len(self.colors)
         return(result)
 
+    @staticmethod
+    def weighted_quantiles(values, quantiles, sample_weight=None):
```

Comment:
Naming is a bit weird. Suggest ``values`` and ``weights``, or ``samples`` and ``sample_weights``. Former seems shorter :smile: